### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/how-to/index.md` and `docs/zh/docs/how-to/general.md`

### PR DESCRIPTION
**Pull request by billzhong**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 191022f52f482f28a80d89044acd5200a22ed7d9 at: https://21640f9f.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 528a5a9196c83e5ffe73bb974aa22f7bd1a67349 at: https://e60448e3.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8b81a4988acf9a9177fcd7068dae9767111279b2 at: https://ac4f2875.fastapitiangolo.pages.dev

**Comment by waketzheng** [comment]
原文为`How To - Recipes`
翻译成了`如何操作 - 食谱`
感觉怪怪的，Recipes译成`秘诀`或`诀窍`，是不是更贴切？

**Comment by tiangolo** [comment]
📝 Docs preview for commit 924c56ff8ff202f885d5576c682c6c02b7ef1be7 at: https://c32fbe0a.fastapitiangolo.pages.dev

File: docs/zh/docs/how-to/general.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# 通用 - 如何操作 - 诀窍
+
+这里是指向文档中其他部分的一些链接，用于解答一般性或常见问题。
```
Comment (nick-cjyx9) [inline docs/zh/docs/how-to/general.md:3]: 应改为：
一些指向文档中其他部分的链接

ref: [多层定语顺序](https://baike.baidu.com/item/%E5%A4%9A%E5%B1%82%E5%AE%9A%E8%AF%AD%E9%A1%BA%E5%BA%8F/4483121)

File: docs/zh/docs/how-to/general.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## 数据过滤 - 安全性
+
+为确保不返回超过需要的数据，请阅读 [教程 - 响应模型 - 返回类型](../tutorial/response-model.md){.internal-link target=_blank} 文档。
+
+## 文档的标签 - OpenAPI
+
+在文档界面中添加*路径操作*的标签和进行分组，请阅读 [教程 - 路径操作配置 - Tags 参数](../tutorial/path-operation-configuration.md#tags){.internal-link target=_blank} 文档。
```
Comment (nick-cjyx9) [inline docs/zh/docs/how-to/general.md:11]: 中文文本中通常不使用斜体，该用加粗

ref: [簡單做好中文排版](https://wanderer.tw/簡單做好中文排版/)第五项原则

Comment (nick-cjyx9) [reply docs/zh/docs/how-to/general.md:11]: 15, 21, 23行同

**Review by nick-cjyx9 — CHANGES_REQUESTED** [review]

**Review by nick-cjyx9 — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit a5c774e105487e4014f995e787d7577e5da68b9f at: https://c3c132c6.fastapitiangolo.pages.dev

**Comment by nick-cjyx9** [comment]
@waketzheng Do you have time to review these changes?

**Review by waketzheng — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for the contribution! @billzhong  :nerd_face: 

And thanks for the reviews @waketzheng @nick-cjyx9 :smile_cat: :rocket:

**Pull request merged**